examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -88,15 +88,15 @@
 #############################################################################
 # Obtain prediction scores via cross validation
 # -----------------------------------------------
-from sklearn.cross_validation import LeaveOneLabelOut, cross_val_score
+from sklearn.model_selection import LeaveOneGroupOut, cross_val_score
 
 # Define the cross-validation scheme used for validation.
-# Here we use a LeaveOneLabelOut cross-validation on the session label
+# Here we use a LeaveOneGroupOut cross-validation on the session group
 # which corresponds to a leave-one-session-out
-cv = LeaveOneLabelOut(session)
+cv = LeaveOneGroupOut()
 
 # Compute the prediction accuracy for the different folds (i.e. session)
-cv_scores = cross_val_score(anova_svc, X, conditions, cv=cv)
+cv_scores = cross_val_score(anova_svc, X, conditions, cv=cv.split(X=X, groups=session))
```

Comment:
This is not the right fix: you need to pass "groups=session" to cross_val_score and leave cv unchanged.

Reply:
Okay, I'll check that out. That would also obviate the SearchLight issue that I filed.